README.md

```diff
@@ -127,3 +127,35 @@ management:
 Documentation of exposed services is available at:
     
     localhost:5080/swagger-ui.html
+
+
+## Developpment Environment
```

Comment:
This is just a proposal to make development easier, only needing to launch a docker container to develop, test or build the project (no need to install java, simple to change java version or other language...)